### ISSUE TITLE
revoking licenses

### ISSUE THREAD

**Issue by buckettttt** [gitrous/dtc#14]
give license checkers (or police) the ability to revoke licenses incase they accidentally accepted one they didn't mean to accept or (in this case there's no police) see someone misusing their license but can't take legal action.

**Comment by xXPirateLordXx** [comment]
I don't agree with the police part, if police could then some abusive trainee might take it away for no reason or for having a misdemeanor.

**Comment by lovegamesdude** [comment]
Ok well trainees cant arrest people, take PD guns or PD tazers so I don't know why you think it's impossible to add a feature where the trainees can't revoke permits. Also I'm pretty sure it's an automatic feature that you lose your license when you get arrested

**Comment by xXPirateLordXx** [comment]
oh i didnt know trainees couldnt arrest i thought i'd seen them with cuffs before but ok,
